bridge/test/Federation_test.js

```diff
@@ -342,76 +342,38 @@ contract('FederationV2', async function (accounts) {
 
                 it('should be succesful', async function() {
                     const receipt = await this.federators.emitHeartbeat(
+                        fedRskBlock,
+                        fedEthBlock,
                         federatorVersion,
-                        [ fedRskChainId, fedEthChainId, fedBscChainId ],
-                        [ fedRskBlock, fedEthBlock, fedBscBlock ],
-                        [ nodeRskInfo, nodeEthInfo, nodeBscInfo ],
+                        nodeRskInfo,
+                        nodeEthInfo,
                         {from: federator1}
                     );
                     utils.checkRcpt(receipt);
 
                     assert.equal(receipt.logs[0].event, 'HeartBeat');
                     assert.equal(receipt.logs[0].args[0], federator1);
-                    assert.equal(receipt.logs[0].args[1], await web3.eth.net.getId() );
-                    assert.equal(receipt.logs[0].args[2], await web3.eth.getBlockNumber());
+                    assert.equal(receipt.logs[0].args[1], fedRskBlock );
+                    assert.equal(receipt.logs[0].args[2], fedEthBlock);
                     assert.equal(receipt.logs[0].args[3], federatorVersion);
-                    assert.deepEqual(arrayMembersToString(receipt.logs[0].args[4]),
-                        [ fedRskChainId, fedEthChainId, fedBscChainId ]
-                    );
-                    assert.deepEqual(arrayMembersToString(receipt.logs[0].args[5]), [ fedRskBlock, fedEthBlock, fedBscBlock ]);
-                    assert.deepEqual(arrayMembersToString(receipt.logs[0].args[6]), [ nodeRskInfo, nodeEthInfo, nodeBscInfo ]);
+                    assert.equal(receipt.logs[0].args[4], nodeRskInfo);
+                    assert.equal(receipt.logs[0].args[5], nodeEthInfo);
                 });
 
                 it('should fail if not a memeber', async function() {
                     await truffleAssertions.fails(
                         this.federators.emitHeartbeat(
+                            fedRskBlock,
+                            fedEthBlock,
                             federatorVersion,
-                            [ fedRskChainId, fedEthChainId, fedBscChainId ],
-                            [ fedRskBlock, fedEthBlock, fedBscBlock ],
-                            [ nodeRskInfo, nodeEthInfo, nodeBscInfo ],
+                            nodeRskInfo,
+                            nodeEthInfo,
                             {from: anAccount}
                         ),
                         truffleAssertions.ErrorType.REVERT
                     );
                 });
-
-                it('should fail if different array size', async function() {
```

Comment:
can you explain why this test is removed? (just to have context)

Reply:
Because there is no array parameter in the contract more, there was in a old version.